en/asgardeo/docs/apis/restapis/actions.yaml

```diff
@@ -0,0 +1,859 @@
+      flows:
+        authorizationCode:
+          authorizationUrl: 'https://api.asgardeo.io/t/{org-name}/oauth2/authorize'
+          tokenUrl: 'https://api.asgardeo.io/t/{org-name}/oauth2/token'
+          scopes: 
+            read: internal_action_mgt_view
+            write: internal_action_mgt_create
+            update: internal_action_mgt_update
+            delete: internal_action_mgt_delete
+
+  schemas:
+    ActionModel:
+      type: object
+      required:
+        - name
+        - endpoint
+      properties:
+        name:
+          type: string
+          example: Access Token Pre Issue
+          minLength: 1
+          maxLength: 255
+        description:
+          type: string
+          example: This is the configuration of pre-action for issuing access token.
+          maxLength: 255
+        endpoint:
+          $ref: '#/components/schemas/Endpoint'
+
+    Endpoint:
+      type: object
+      required:
+        - uri
+        - authentication
+      properties:
+        uri:
+          type: string
+          example: https://myextension.com/token
+          pattern: '^https?://.+'
+        authentication:
+          $ref: '#/components/schemas/Authentication'
+
+    EndpointResponse:
+      type: object
+      required:
+        - uri
+        - authentication
+      properties:
+        uri:
+          type: string
+          example: https://myextension.com/token
+          pattern: '^https?://.+'
+        authentication:
+          $ref: '#/components/schemas/AuthenticationTypeResponse'
+
+    ActionResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          example: '24f64d17-9824-4e28-8413-de45728d8e84'
+        type:
+          type: string
+          enum:
+            - PRE_ISSUE_ACCESS_TOKEN
+            - PRE_UPDATE_PASSWORD
+            - PRE_UPDATE_PROFILE
+            - PRE_REGISTRATION
+        name:
+          type: string
+          example: Access Token Pre Issue
+          minLength: 1
+          maxLength: 255
+        description:
+          type: string
+          example: This is the configuration of pre-action for issuing access token.
+          maxLength: 255
+        status:
+          type: string
+          enum:
+            - ACTIVE
+            - INACTIVE
+        endpoint:
+          $ref: '#/components/schemas/EndpointResponse'
+
+    ActionBasicResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          example: '24f64d17-9824-4e28-8413-de45728d8e84'
+        type:
+          type: string
+          enum:
+            - PRE_ISSUE_ACCESS_TOKEN
+            - PRE_UPDATE_PASSWORD
+            - PRE_UPDATE_PROFILE
+            - PRE_REGISTRATION
+        name:
+          type: string
+          example: Access Token Pre Issue
+          minLength: 1
+          maxLength: 255
+        description:
+          type: string
+          example: This is the configuration of pre-action for issuing access token.
```

Comment:
Fix based on comment above

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4698/commits/ee4f5ca7c24d3d8dcaca4943b7b5aadb6e16e7fc